src/standalone/Vegetation/PlantHydraulics.jl

```diff
@@ -63,48 +67,22 @@ transpiration (Prescribed or Diagnostic)
 """
 abstract type AbstractTranspiration{FT <: AbstractFloat} end
 
-
 """
-    PlantHydraulicsParameters{FT <: AbstractFloat}
+   PrescribedSiteAI{FT <:AbstractFloat}
+
+A struct containing the area indices of the plants at a specific site;
+LAI varies in time, while SAI and RAI are fixed. No spatial variation is
+modeled.
 
-A struct for holding parameters of the PlantHydraulics Model.
 $(DocStringExtensions.FIELDS)
 """
-struct PlantHydraulicsParameters{FT <: AbstractFloat, CP, RP}
-    "RAI, SAI, LAI in [m2 m-2]"
-    area_index::NamedTuple{(:root, :stem, :leaf), Tuple{FT, FT, FT}}
-    "porosity (m3/m3)"
-    ν::FT
-    "storativity (m3/m3)"
-    S_s::FT
-    "Root distribution function P(z)"
-    root_distribution::Function
-    "Conductivity model and parameters"
-    conductivity_model::CP
-    "Water retention model and parameters"
-    retention_model::RP
-end
-
-function PlantHydraulicsParameters(;
-    area_index::NamedTuple{(:root, :stem, :leaf), Tuple{FT, FT, FT}},
-    ν::FT,
-    S_s::FT,
-    root_distribution::Function,
-    conductivity_model,
-    retention_model,
-) where {FT}
-    return PlantHydraulicsParameters{
-        FT,
-        typeof(conductivity_model),
-        typeof(retention_model),
-    }(
-        area_index,
-        ν,
-        S_s,
-        root_distribution,
-        conductivity_model,
-        retention_model,
-    )
+struct PrescribedSiteAI{FT <: AbstractFloat}
```

Comment:
eventually inherits from  "AbstractPrescribedFunction" type? which would encompass any prescribed parameter function?

ties into what we were discussing yesterday @juliasloan25 - but too soon to implement now, I think, until we also have the different examples from the bucket work

Reply:
I agree :)